sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -316,73 +316,31 @@
         </executions>
       </plugin>
 
-      <!-- Performs test coverage analysis -->
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
         <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
         <executions>
           <execution>
-            <id>default-instrument</id>
+            <id>prepare-agent</id>
             <goals>
-              <goal>instrument</goal>
+              <goal>prepare-agent</goal>
             </goals>
           </execution>
           <execution>
-            <id>default-restore-instrumented-classes</id>
+            <id>default-instrument</id>
             <goals>
-              <goal>restore-instrumented-classes</goal>
+              <goal>instrument</goal>
             </goals>
           </execution>
           <execution>
-            <!-- This generates the unit test reports for individual modules.
-              jacoco-test-coverage generates aggregate reports for all modules -->
-            <id>post-unit-test</id>
-            <goals>
-              <goal>report</goal>
-            </goals>
-            <configuration>
-              <!-- Sets the output directory for the code coverage report. -->
-              <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
-              <excludes>
-                <exclude>**/com/azure/cosmos/implementation/apachecommons/**/*</exclude>
-                <exclude>**/com/azure/cosmos/implementation/guava25/**/*</exclude>
-                <exclude>**/com/azure/cosmos/implementation/guava27/**/*</exclude>
-              </excludes>
-            </configuration>
-          </execution>
-          <execution>
-            <id>check</id>
+            <id>default-restore-instrumented-classes</id>
             <goals>
-              <goal>check</goal>
+              <goal>restore-instrumented-classes</goal>
             </goals>
-            <configuration>
-              <rules>
-                <rule>
-                  <element>BUNDLE</element>
-                  <limits>
-                    <limit>
-                      <counter>LINE</counter>
-                      <value>COVEREDRATIO</value>
-                      <minimum>${jacoco.min.linecoverage}</minimum>
-                    </limit>
-                    <limit>
-                      <counter>BRANCH</counter>
-                      <value>COVEREDRATIO</value>
-                      <minimum>${jacoco.min.branchcoverage}</minimum>
-                    </limit>
-                  </limits>
-                </rule>
-              </rules>
```

Comment:
This is required to ensure we don't fall below coverage threshold. The PR should fail if the coverage is below this threshold.